addons/nuxeo-csv/nuxeo-csv-core/src/test/java/org/nuxeo/ecm/csv/core/operation/TestImportOperation.java

```diff
@@ -142,4 +143,53 @@ public void testImportOperation() throws OperationException, InterruptedExceptio
         assertEquals(336, result.getSuccessLineCount());
         assertEquals(336, result.getTotalLineCount());
     }
+    
+    @Test
+    public void TestImportIgnoreSurroundingSpaces() throws OperationException, InterruptedException {
+    	Map<String, Object> params = new HashMap<>();
+        params.put("path", testFolder.getPathAsString());
```

Comment:
You could use `Map<String, Object> params = Map.of("path", testFolder.getPathAsString());`